dev-util/sccache/metadata.xml

```diff
@@ -11,6 +11,7 @@
 		<flag name="gcs">Enable Google Cloud Storage support</flag>
 		<flag name="redis">Enable Redis support</flag>
 		<flag name="s3">Enable Amazon S3 support</flag>
+		<flag name="simple-s3">Enable Amazon S3 support without s3</flag>
```

Comment:
description is rather confusing =)
let's do something `Enable minimal Amazon S3 support`
and change other one to `Enable full Amazon S3 support`

Reply:
Yes, that is better and I changed it. I wish upstream would of documented the feature so I had a better idea how it differed from the standard `s3` support.